R/Connect.R

```diff
@@ -82,6 +82,57 @@ createConnectionDetails <- function(dbms,
 }
 
 jdbcDrivers <- new.env()
+jdbcDrivers$defaultPath <- Sys.getenv("DATABASECONNECTOR_JAR_FOLDER", 
+                                      unset = path.expand("~/DatabaseConnectorDrivers"))
+
+#' Download DatabaseConnector JDBC Jar files
+#' 
+#' Download the DatabaseConnector JDBC drivers from https://github.com/OHDSI 
+#'
+#' @param destination The full path to the folder where the Jar files should be downloaded to.
+#'        If NULL (default) then the value of the environment variable "DATABASECONNECTOR_JAR_FOLDER" is used.
+#'        If DATABASECONNECTOR_JAR_FOLDER is not set then "~/DatabaseConnectorDrivers" is used.
+#' @param method The method used for downloading files. See \code{?download.file} for details and options.
+#' @param ... Further arguments passed on to \code{download.file} 
+#' 
+#' @return Invisibly returns the destination if the download was successful.
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' downloadJdbcDrivers()
+#' }
+downloadJdbcDrivers <- function(destination = NULL, method = "auto", ...){
+  if (is.null(destination) || missing(destination)) {
+    destination <- jdbcDrivers$defaultPath
+  }
+  
+  if (!dir.exists(destination)) dir.create(destination)
+  
+  baseUrl <- "https://github.com/OHDSI/DatabaseConnectorJars/blob/master/inst/java"
+  jdbcDriverNames <- c("RedshiftJDBC4-no-awssdk-1.2.20.1043.jar",
+                       "ojdbc8.jar",
+                       "postgresql-42.2.5.jre6.jar",
+                       "sqljdbc4.jar")
+  
```

Comment:
RedShift now requires a [ton of jars](https://github.com/OHDSI/DatabaseConnectorJars/tree/develop/inst/java) to be installed (which is why we're being refused by CRAN). We need to install all of those (which also argues for allowing users to specify which driver to install).

Reply:
![image](https://user-images.githubusercontent.com/10227522/100622608-0e18a200-32ef-11eb-8c06-5a9e02bb102b.png)
Ok. I'm assuming all of the new jar files are for Redshift. Maybe we should have a readme file in this folder explaining what the files are and where they came from.

Reply:
The user can specify which dbms they need drivers for.